jest.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  preset: "ts-jest",
+  testEnvironment: "node",
+  roots: ["<rootDir>/lib"],
+  testMatch: ["**/tests/**/*.test.ts"],
```

Comment:
Jest will find all test files that live under a folder called `tests` and have an extention of `.test.ts`.

For example:
* `/lib/tests/parser.test.ts` 👍 
* `/lib/parser.spec.ts` 👎